browser/ui/webui/settings/brave_appearance_handler.cc

```diff
@@ -7,11 +7,34 @@
 
 #include "base/bind.h"
 #include "base/strings/string_number_conversions.h"
+#include "brave/browser/ntp_background_images/view_counter_service_factory.h"
 #include "brave/browser/themes/brave_dark_mode_utils.h"
 #include "brave/common/pref_names.h"
+#include "brave/components/ntp_background_images/browser/ntp_background_images_data.h"
+#include "brave/components/ntp_background_images/browser/view_counter_service.h"
+#include "brave/components/ntp_background_images/common/pref_names.h"
 #include "chrome/browser/browser_process.h"
+#include "chrome/browser/profiles/profile.h"
 #include "content/public/browser/web_ui.h"
 
+using ntp_background_images::ViewCounterServiceFactory;
+using ntp_background_images::prefs::kNewTabPageSuperReferralThemesOption;
+
+namespace {
+
+bool IsSuperReferralActive(Profile* profile) {
+  bool isSuperReferralActive = false;
+  if (auto* service = ViewCounterServiceFactory::GetForProfile(profile)) {
+    auto* data = service->GetCurrentBrandedWallpaperData();
+    if (data && data->IsSuperReferral())
```

Comment:
nit: better to use braces for consistency here?

Reply:
fixed.